packages/telemetry/example/index.ts

```diff
@@ -0,0 +1,49 @@
+import { initTelemetrySettings, ClientFactory } from '../src';
+import 'dotenv/config';
+
+/**
+ * Helper function to detect if env var is provided before returning it.
+ *
+ * @param name Environment variable name
+ * @returns Environment variable value
+ */
+function env(name: string): string {
+    if (process.env[name]) {
+        return process.env[name] as string;
+    } else {
+        throw new Error(`Env var ${name} is required`);
+    }
+}
+
+/**
+ * Example function of sending a telemetry event.
+ */
+async function sendTelemetryEvent(): Promise<void> {
+    console.log('Reporting...');
```

Comment:
We should be using the logger and not console logs

Reply:
It is an example outside src folder. I will switch to use @sap-ux/logger